plugins/context_datalayer_reaction.inc

```diff
@@ -162,11 +162,14 @@ class context_datalayer_reaction extends context_reaction {
     foreach ($this->get_contexts() as $context) {
       $options = $this->fetch_from_context($context);
       if (!empty($options['data'])) {
+        $data = array_map(function($value) {
+          return token_replace($value);
+        }, $options['data']);
```

Comment:
I was going to just use `$data = array_map('token_replace', $options['data']);` but then I wondered whether we should pass a node object in?  Thought I'd wait for some feedback first before adding that part (in case it's not needed?) ...